packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -58,6 +71,22 @@ export class FauxClassGenerator {
     ['complexvalue', 'Object']
   ]);
 
+  constructor(emitter: EventEmitter, cancellationToken?: CancellationToken) {
+    this.emitter = emitter;
+    this.cancellationToken = cancellationToken;
+  }
+
+  private errorExit(errorMessage: string): Promise<string> {
+    this.emitter.emit(LocalCommandExecution.STDERR_EVENT, errorMessage);
+    this.emitter.emit(LocalCommandExecution.ERROR_EVENT, '1');
```

Comment:
Do you properly conform to the events from Node.js? The error event is supposed to emit an `Error` object and not a string, like what you have done. See https://nodejs.org/api/child_process.html#child_process_event_error

Reply:
I'll go take a look - good observation.  At some level, the emitter events are arbitrarily defined by the emitter - but best to conform to standard patterns.